### PR TITLE
feat: added industry section on the bottom

### LINKED ISSUE #83: Industry Section
Related to #78 

Image

### PR DESCRIPTION
**Pull request by morshlab**
# Description

Added industry section with named `akij-turborepo #83`

Fixes #83  (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

**Test Configuration**:
* Firmware version:
* Hardware:
* Toolchain:
* SDK:

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
image

### CODE REVIEW

File: apps/akij-food-beverage/src/components/ui/features/IndustrySection.tsx [hunk 1]
```diff
@@ -0,0 +1,53 @@
+import {useEffect, useState} from 'react'
+import CheckmarkIcon from "ui/components/elements/Icon/checkmark";
+import { fetchProducts } from '../../../utils/fetchProducts';
+import { urlForThumbnail } from './../../../utils/imageProcess'
+const Industry = () => {
```
Comment (TanvirAlam) [inline apps/akij-food-beverage/src/components/ui/features/IndustrySection.tsx:5]: please add empty space before

File: apps/akij-food-beverage/src/components/ui/features/IndustrySection.tsx [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+          try {
+            const products = await fetchProducts()
+            console.log(products)
+            setState(products[4])
+          } catch (e) {
+            console.log(e)
+          }
+        }
+        fetchData()
+      }, [])
```
Comment (TanvirAlam) [inline apps/akij-food-beverage/src/components/ui/features/IndustrySection.tsx:18]: please add empty space

File: apps/akij-food-beverage/src/components/ui/features/IndustrySection.tsx [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+                     <span>Resonable Price</span>
+                    </div>
+                    <p>Lorem, ipsum dolor sit amet consectetur adipisicing elit. Optio explicabo soluta sapiente fugiat magni ipsa, neque delectus architecto </p>
+                </li>
+            </ul>
+        </div>
+    </div>
+  )
+}
+export default Industry;
```
Comment (TanvirAlam) [inline apps/akij-food-beverage/src/components/ui/features/IndustrySection.tsx:53]: please add new empty space

File: apps/akij-food-beverage/src/styles/globals.css [hunk 4]
```diff
@@ -1,3 +1,23 @@
...
+    .outlineBtn {
+      @apply text-center text-blue-700 rounded-full hover:shadow-md hover:shadow-[#0C66EE]/50 transition duration-300 w-full sm:max-w-[240px] px-6 py-4 border border-[#0C66EE];
+    }
+    .brand{
+        @apply text-transparent bg-clip-text bg-gradient-to-r from-blue-400 to-blue-600;
+    }
+    .description{
+        @apply my-8 leading-7 tracking-wide text-slate-700;
+    }
+  }
```
Comment (TanvirAlam) [inline apps/akij-food-beverage/src/styles/globals.css:23]: please add empty new line

File: packages/ui/components/elements/Icon/checkmark.tsx [hunk 5]
```diff
@@ -0,0 +1,10 @@
+const Checkmark = () => {
+    return (
+      <span aria-hidden="true" role="img" className="material-design-icon check-circle-icon text-[#0C66EE]">
+      <svg fill="currentColor" width="20" height="20" viewBox="0 0 24 24" className="material-design-icon__svg">
+          <path d="M12 2C6.5 2 2 6.5 2 12S6.5 22 12 22 22 17.5 22 12 17.5 2 12 2M10 17L5 12L6.41 10.59L10 14.17L17.59 6.58L19 8L10 17Z">
+          </path>
+      </svg>
+  </span>   )
+  }
+  export default  Checkmark;
```
Comment (TanvirAlam) [inline packages/ui/components/elements/Icon/checkmark.tsx:10]: please add empty new line

File: apps/akij-food-beverage/src/components/ui/features/IndustrySection.tsx [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+                      <span>Natural Flavour </span>
+                   </div>
+                   <p>Lorem, ipsum dolor sit amet consectetur adipisicing elit. Optio explicabo soluta sapiente fugiat magni ipsa, neque delectus architecto </p>
+                </li>
+                <li className="space-y-2">
+                   <div className="flex items-center space-x-2">
+                     <CheckmarkIcon />
+                     <span>Country Side</span>
+                   </div>
+                   <p>Lorem, ipsum dolor sit amet consectetur adipisicing elit. Optio explicabo soluta sapiente fugiat magni ipsa, neque delectus architecto </p>
```
Comment (TanvirAlam) [inline apps/akij-food-beverage/src/components/ui/features/IndustrySection.tsx:39]: why dnt you pass this as string? we have rule, NO static string in templates

File: apps/akij-food-beverage/src/components/ui/features/IndustrySection.tsx [hunk 7]
```diff
@@ -0,0 +1,53 @@
...
+                </li>
+                <li className="space-y-2">
+                   <div className="flex items-center space-x-2">
+                     <CheckmarkIcon />
+                     <span>Resonable Price</span>
+                    </div>
+                    <p>Lorem, ipsum dolor sit amet consectetur adipisicing elit. Optio explicabo soluta sapiente fugiat magni ipsa, neque delectus architecto </p>
```
Comment (TanvirAlam) [inline apps/akij-food-beverage/src/components/ui/features/IndustrySection.tsx:46]: why dont you pass this as string? we have rule, NO static string in templates

File: apps/akij-food-beverage/src/components/ui/features/IndustrySection.tsx [hunk 8]
```diff
@@ -0,0 +1,53 @@
...
+        </div>
+        <div className='my-24 mx-auto'>
+            <h2 className="text-4xl font-semibold"> Frutika Pure Mango <br />Taste Original</h2>
+            <ul className="space-y-4 sm:space-y-2">
+                <li className="space-y-2">
+                   <div className="flex items-center space-x-2">
+                      <CheckmarkIcon />
+                      <span>Natural Flavour </span>
+                   </div>
+                   <p>Lorem, ipsum dolor sit amet consectetur adipisicing elit. Optio explicabo soluta sapiente fugiat magni ipsa, neque delectus architecto </p>
```
Comment (TanvirAlam) [inline apps/akij-food-beverage/src/components/ui/features/IndustrySection.tsx:32]: why dnt you pass this as string? we have rule, NO static string in templates

File: apps/akij-food-beverage/src/components/ui/features/IndustrySection.tsx [hunk 9]
```diff
@@ -0,0 +1,53 @@
+import {useEffect, useState} from 'react'
+import CheckmarkIcon from "ui/components/elements/Icon/checkmark";
+import { fetchProducts } from '../../../utils/fetchProducts';
+import { urlForThumbnail } from './../../../utils/imageProcess'
+const Industry = () => {
+  const [state, setState] = useState({})
+    useEffect(() => {
```
Comment (TanvirAlam) [inline apps/akij-food-beverage/src/components/ui/features/IndustrySection.tsx:7]: why do you have this hook? if you are not using it at all?

File: apps/akij-food-beverage/src/components/ui/features/IndustrySection.tsx [hunk 10]
```diff
@@ -0,0 +1,53 @@
...
+                   <div className="flex items-center space-x-2">
+                     <CheckmarkIcon />
+                     <span>Country Side</span>
+                   </div>
+                   <p>Lorem, ipsum dolor sit amet consectetur adipisicing elit. Optio explicabo soluta sapiente fugiat magni ipsa, neque delectus architecto </p>
+                </li>
+                <li className="space-y-2">
+                   <div className="flex items-center space-x-2">
+                     <CheckmarkIcon />
+                     <span>Resonable Price</span>
```
Comment (TanvirAlam) [inline apps/akij-food-beverage/src/components/ui/features/IndustrySection.tsx:44]: again static string

File: apps/akij-food-beverage/src/components/ui/features/IndustrySection.tsx [hunk 11]
```diff
@@ -0,0 +1,53 @@
...
+    <div className="customContainer gridTwo">
+        <div className='my-16 mx-auto'>
+           <img src={state.image && urlForThumbnail(state.image)} alt={state.name} height={400} width={300}/>
+        </div>
+        <div className='my-24 mx-auto'>
+            <h2 className="text-4xl font-semibold"> Frutika Pure Mango <br />Taste Original</h2>
+            <ul className="space-y-4 sm:space-y-2">
+                <li className="space-y-2">
+                   <div className="flex items-center space-x-2">
+                      <CheckmarkIcon />
```
Comment (TanvirAlam) [inline apps/akij-food-beverage/src/components/ui/features/IndustrySection.tsx:29]: you are using this twice, why not make it into a components, you calling this twice or 3 times?

```
<div className="flex items-center space-x-2">
   <CheckmarkIcon />
   <span>Country Side</span>
 </div>
 ```

**Review by TanvirAlam — CHANGES_REQUESTED** [review]

**Comment by TanvirAlam** [comment]
LGTM, fix your merge and do the changes and push for approval

**Comment by TanvirAlam** [comment]
@morshlab Make the Image bigger, look at the padding, fix them you are our CFO, yo? was up men

File: apps/akij-food-beverage/src/components/ui/features/IndustrySection.tsx (hunk 1 above)
Comment (urmirahman) [reply apps/akij-food-beverage/src/components/ui/features/IndustrySection.tsx:5]: Empty space added

File: apps/akij-food-beverage/src/components/ui/features/IndustrySection.tsx (hunk 2 above)
Comment (urmirahman) [reply apps/akij-food-beverage/src/components/ui/features/IndustrySection.tsx:18]: Empty space added

File: apps/akij-food-beverage/src/components/ui/features/IndustrySection.tsx (hunk 7 above)
Comment (urmirahman) [reply apps/akij-food-beverage/src/components/ui/features/IndustrySection.tsx:46]: Creating new component for that

File: apps/akij-food-beverage/src/components/ui/features/IndustrySection.tsx (hunk 8 above)
Comment (urmirahman) [reply apps/akij-food-beverage/src/components/ui/features/IndustrySection.tsx:32]: creating new component for that

File: apps/akij-food-beverage/src/components/ui/features/IndustrySection.tsx (hunk 9 above)
Comment (urmirahman) [reply apps/akij-food-beverage/src/components/ui/features/IndustrySection.tsx:7]: useState is been used.

File: apps/akij-food-beverage/src/components/ui/features/IndustrySection.tsx (hunk 10 above)
Comment (urmirahman) [reply apps/akij-food-beverage/src/components/ui/features/IndustrySection.tsx:44]: making new component

File: apps/akij-food-beverage/src/components/ui/features/IndustrySection.tsx (hunk 11 above)
Comment (urmirahman) [reply apps/akij-food-beverage/src/components/ui/features/IndustrySection.tsx:29]: making new component

**Comment by urmirahman** [comment]
@TanvirAlam  All conflicts has been gone. please check.

**Review by TanvirAlam — APPROVED** [review]
LGTM :+1: well done @urmirahman , yes try changing the them color to see the effect.

**Pull request merged**